openssl-dynamic/src/main/c/ssl.c

```diff
@@ -106,7 +106,7 @@ struct TCN_bio_bytebuffer {
  * so that the return value for SSL_OP_FUTURE_WORKAROUND will only be
  * reported by versions that actually support that specific workaround.
  */
-static const jint supported_ssl_opts = 0
+static const jlong supported_ssl_opts = 0
```

Comment:
my preference is to remove all this code and the `hasOp` method. The `hasOp` method is not used in Netty, this code is likely incomplete at compile time, and  is very likely inaccurate if dynamic linking is used.